web/config/jest-setup.ts

```diff
@@ -0,0 +1,2 @@
+import "intersection-observer";
+import "@testing-library/jest-dom";
```

Comment:
There's a whole set of things which aren't pollyfilled by `jsdom` which is what the `@testing-library/jest-dom` is for. We also need to pollyfill the `intersection-observer` for the `AppConfig` component to render.